### PR TITLE
Updating README for 1.11 release

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-**NOTE**: This is a preliminary implementation of the [FIDO IoT Spec - Working Draft ](https://fidoalliance.org/specs/fidoiot/FIDO-IoT-spec-v1.0-wd-20200730.html) published by the FIDO Alliance. The implementation is experimental and incomplete, and is not ready for use in any production capacity. Some cryptographic algorithms and encoding formats have not been implemented, and any aspect of this implementation is subject to change.
+**NOTE**: This is a preliminary implementation of the [FIDO IoT Spec](https://fidoalliance.org/specs/FDO/fido-device-onboard-v1.0-ps-20210323/) published by the FIDO Alliance. The implementation is experimental and incomplete, and is not ready for use in any production capacity. Some cryptographic algorithms and encoding formats have not been implemented, and any aspect of this implementation is subject to change.
```
Comment (trbehera) [inline README.md:1]: FIDO IoT Spec => FIDO Device Onboard Spec

File: README.md [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
+- `http-api-samples`: It contains Servlet implementation for various operations to be performed using different REST endpoints for all server service.
 
-- `service`: It contains implementations for FDO HTTP servers and clients.
+- `protocol-samples`: It contains client and server implementation that demonstrates the prelimnary E2E demo that educates the enduser with the protocol workflow.
 
-- `serviceinfo`: It contains implementations for Owner and Device ServiceInfo.
+- `protocol`: It contains implementations related to protocol message processing..
 
-- `storage`: It contains SQL based storage implementations for FDO PRI Servers.
+- `util`: It contains utility package such as storage, serviceinfo, dispatchers - for message passing and cert-utils - loading certificates and keys from PEM formatted strings.
```
Comment (trbehera) [inline README.md:26]: serviceinfo => ServiceInfo

File: component-samples/demo/README.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ In the following use-case (use-case: 1), the FDO PRI Manufacturer provisions the
 
 3. Complete Ownership Voucher Extension by using the API `GET /api/v1/vouchers/<serial_no>` and save the Ownership Voucher. Assign customer with customer Id '2' to the device. Please refer to [FDO PRI Manufacturer REST APIs](manufacturer/README.md/#fdo-pri-manufacturer-rest-apis) for more information about assigning customer to a device.
 
-4. Start the FDO PRI Reseller Sample as per the steps outlined in [Reseller README](reseller/README.md).
+4. After DI and before starting Transfer Ownership Protocol 1, add the owner2 keypair in the current owner's (reseller) keystore. Start the FIDO IoT Reseller Sample as per the steps outlined in [Reseller README](reseller/README.md).
```
Comment (trbehera) [inline component-samples/demo/README.md:130]: FIDO IoT Reseller => FDO PRI Reseller

File: component-samples/demo/manufacturer/README.md [hunk 4]
```diff
@@ -10,17 +10,17 @@
 
 Use the following commands to build FIDO Device Onboard (FDO) Protocol Reference Implementation
 (PRI) Manufacturer component sample source.
-For the instructions in this document, `<pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
+For the instructions in this document, `<fdo-pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
 ```
 $ cd <pri-src>/component-samples/manufacturer/
```
Comment (trbehera) [inline component-samples/demo/manufacturer/README.md:15]: pri-src => fdo-pri-src

File: component-samples/demo/manufacturer/README.md [hunk 5]
```diff
@@ -76,6 +76,33 @@ Some required runtime arguments
...
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: 443
+
+- `manufacturer_ssl_keystore`
+
+  Provides path for SSL keystore to be used by the service, in case it runs in HTTPS mode.
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: <fido-iot-src>/component-samples/demo/manufacturer/certs/ssl.p12
```
Comment (trbehera) [inline component-samples/demo/manufacturer/README.md:97]: fido-iot-src => fdo-pri-src

File: component-samples/demo/owner/README.md [hunk 6]
```diff
@@ -8,17 +8,17 @@
 # Getting the executable
 
 Use the following commands to build FIDO Device Onboard (FDO) Owner Component sample source.
-For the instructions in this document, `<pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
+For the instructions in this document, `<fdo-pri-src>` refers to the path of the FDO PRI folder 'pri-fidoiot'.
 ```
 $ cd <pri-src>/component-samples/owner/
```
Comment (trbehera) [inline component-samples/demo/owner/README.md:13]: pri-src => fdo-pri-src

File: component-samples/demo/owner/README.md [hunk 7]
```diff
@@ -113,17 +113,58 @@ Some required runtime arguments
...
+  There are 2 modes namely 'TEST' and 'PROD'. TEST mode disables SSL verification.
+
+  Default value: TEST
+
+- `owner_ssl_keystore`
+
+  Provides path for SSL keystore to be used by the service, in case it runs in HTTPS mode.
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: <fido-iot-src>/component-samples/demo/owner/certs/ssl.p12
```
Comment (trbehera) [inline component-samples/demo/owner/README.md:140]: fido-iot-src => fdo-pri-src

Also at a few more places.

File: component-samples/demo/reseller/README.md [hunk 8]
```diff
@@ -92,11 +92,38 @@ Some required runtime arguments
...
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: 443
+
+- `reseller_ssl_keystore`
+
+  Provides path for SSL keystore to be used by the service, in case it runs in HTTPS mode.
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: <fido-iot-src>/component-samples/demo/reseller/certs/ssl.p12
```
Comment (trbehera) [inline component-samples/demo/reseller/README.md:113]: fido-iot-src => fdo-pri-src

File: component-samples/demo/rv/README.md [hunk 9]
```diff
@@ -72,6 +72,33 @@ Some required runtime arguments
...
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: 443
+
+- `rv_ssl_keystore`
+
+  Provides path for SSL keystore to be used by the service, in case it runs in HTTPS mode.
+  **NOTE** This property is not required if service is running in `http` mode.
+
+  Default value: <fido-iot-src>/component-samples/demo/rv/certs/ssl.p12
```
Comment (trbehera) [inline component-samples/demo/rv/README.md:93]: fido-iot-src => fdo-pri-src

**Comment by trbehera** [comment]
pri-src => fdo-pri-src
```
README.md:145:"JDBC URL:" = "jdbc:h2:tcp://<localhost>:8049/<pri-src>/protocol-samples/http-server-di-sample/target/data/mfg"
component-samples/demo/manufacturer/README.md:15:$ cd <pri-src>/component-samples/manufacturer/
component-samples/demo/owner/README.md:13:$ cd <pri-src>/component-samples/owner/
```

fido-iot-src => fdo-pri-src
```
component-samples/demo/manufacturer/README.md:97:  Default value: <fido-iot-src>/component-samples/demo/manufacturer/certs/ssl.p12
component-samples/demo/owner/README.md:140:  Default value: <fido-iot-src>/component-samples/demo/owner/certs/ssl.p12
component-samples/demo/owner/README.md:154:  Default value: <fido-iot-src>/component-samples/demo/owner/certs/truststore
component-samples/demo/reseller/README.md:113:  Default value: <fido-iot-src>/component-samples/demo/reseller/certs/ssl.p12
component-samples/demo/rv/README.md:93:  Default value: <fido-iot-src>/component-samples/demo/rv/certs/ssl.p12
```

**Review by trbehera — APPROVED** [review]

**Review by DukeDavis12 — APPROVED** [review]

**Pull request merged**